### DISCUSSION TITLE
Add high speed spinner to GE3

### DISCUSSION THREAD

**Discussion by CaptainJankez** [magknight/787aviatorsTracker#738] [Ideas]
I'm frustrated with the new GEnX engines animation, it looks so ugly, like in the current xcrafts ejets family, you can make a similar animation like in the GE's that were earlier

This is what I'm talking about, https://youtu.be/3Cna0X9vbhc

**Comment by cajunjosh** [comment]
Interesting observation.  I kinda liked the new animations but noticed the spinner graphic was missing from the cone.  Seems like the new animations are a little smoother from startup to idle with better transitions.
